onboarding-workflow/src/actions/emails/dao_invite.ts

```diff
@@ -0,0 +1,53 @@
+import { getDb } from '../../db/db.ts'
+import { INTENT_DAO_INVITE } from './constants.ts'
+
+const EMAIL_SUBJECT = 'You were invited to DAO'
+const EMAIL_HTML_FILE = 'emails/dao_invite.html'
+const EMAIL_TEXT = `\
```

Comment:
I would consider using jinja2 templates and moving e-mail texts into separate files.
dao_invite.html -> dao_invite.html.jinja2
+ dao_invite.text.jinja2